test/object-assign-polyfill.js

```diff
@@ -0,0 +1,33 @@
+// We still run node 0.10 in CI, so include this here for compatibility.
+// Hopefully we can remove this at some point.
+// source: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Object/assign
```

Comment:
I'd be quite happy to get rid of it. Should I remove it from .travis.yml? In that case I'd remove this file too.